### DISCUSSION TITLE
The return error when email already exist

### DISCUSSION THREAD

**Discussion by TheLucDev** [vendurehq/vendure#1964] [Q&A]
hey @michaelbromley , i just write my website base on Vendure , so i want to return an error if email already exist. How can i turn off this guard , i accept the security risk for my website . Thank you so much !

**Comment by giovannefc** [comment]
I've the same problem.

I was going to create a topic about this, but I found the same question here.
I didn't find a way to check if the customer already has the email registered or not.

Can anyone help me about this?

Thanks.
